jest.config.js

```diff
@@ -7,4 +7,7 @@ module.exports = {
   setupFilesAfterEnv: [path.resolve(__dirname, 'jest.setup.js')],
   testEnvironment: 'jsdom',
   testMatch: [path.resolve(__dirname, 'src/**/*.spec.+(ts|tsx)')],
+  moduleNameMapper: {
+    '\\.s?css$': '<rootDir>/src/__mocks__/styleMock.js',
+  },
```

Comment:
You can remove this since we do not have anymore style within `@openfun/verna` package.

Do not forgot to remove `__mocks__/styleMock.js`.